### PR TITLE
TC-388 & TC-441 Landing Page Mobile + 508

### PR DESCRIPTION
**Pull request by kscoulter**
Adjust styles for mobile. 
Replace gradient background.
Make subheadings into h3

### CODE REVIEW

File: src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx [hunk 1]
```diff
@@ -54,27 +54,26 @@ class BreakEvenCalculatorPage extends React.Component {
             investor. It can also be a useful tool in future planning for existing businesses to prove their
             potential turnaround after disaster scenarios.
           </p>
-          <div className={styles.benefitsTitle}>Benefits of a break-even analysis</div>
+          <h2 className={styles.benefitsTitle}>Benefits of a break-even analysis</h2>
           <div className={styles.benefits}>
             {benefitFields.map(benefit => (
               <div className={styles.benefitsCol} key={benefit.title}>
-                <p className={styles.subHeading}>{benefit.title}</p>
+                <h3 className={styles.subHeading}>{benefit.title}</h3>
```
Comment (peggles2) [inline src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx:61]: Nice job changing those for our 508 feedback too!

File: src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.scss [hunk 2]
```diff
@@ -32,7 +32,10 @@
 
       .subHeading {
         color: $secondary-blue;
+        font-size: 1rem;
         font-weight: 700;
+        line-height: 1.5;
+        margin-bottom: 1.1rem;
```
Comment (peggles2) [inline src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.scss:38]: don't we do base spacing for margins?

File: src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.scss [hunk 3]
```diff
@@ -48,15 +51,17 @@
 
   .gradientBackground {
     height: 500px;
-    background: radial-gradient(circle at center, $secondary-blue-light, $secondary-blue-dark);
+    background: $secondary-blue;
```
Comment (peggles2) [inline src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.scss:54]: Nice make sure to close that 508 subtask and put a reference in the Jira ticket that it was taken care of here.

**Review by peggles2 — CHANGES_REQUESTED** [review]
Looks good just one tweak

File: src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.scss (hunk 2 above)
Comment (kscoulter) [reply src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.scss:38]: I took it directly from paragraph styling to keep everything 100% consistent

**Review by peggles2 — APPROVED** [review]

**Pull request merged**